keras/backend/tensorflow_backend.py

```diff
@@ -1239,8 +1242,8 @@ def rnn(step_function, inputs, initial_states,
                 # That's what the tile call does,
                 # it just repeats the mask along its second dimension
                 # n times.
-                tiled_mask_t = tf.tile(mask_t,
-                                       tf.pack([1, tf.shape(output)[1]]))
+                mask_t = tf.squeeze(mask_t)
+                tiled_mask_t = mask_t
```

Comment:
This should be one line.

Either the comment above should be updated, or the code you're proposing is incorrect. Did the behavior of `tf.select` change recently?